src/web3/networkConfig.ts

```diff
@@ -48,6 +48,7 @@ interface NetworkConfig {
   fetchAvailableSuperchargeRewardsV2: string
   resolveId: string
   getPositionsUrl: string
+  getPositionsShortcutsUrl: string
```

Comment:
```suggestion
  getShortcutsUrl: string
```

Reply:
updated! `getShortcutsUrl` was also my first thought but i started to wonder if "shortcuts" is too vague 😅